### DISCUSSION TITLE
[Feature Request] Allow view labels to be reordered

### DISCUSSION THREAD

**Discussion by oguzgelal** [community/community#6796] [Projects and Issues] [answered]
It seems I can't reorder the view labels.

For instance, below I decided to create an `All` view after I created the other two, and I wanted it to be on the very left. It would be nice if it was possible.

<img width="943" alt="Screen Shot 2021-10-27 at 8 34 01 AM" src="https://user-images.githubusercontent.com/2817993/139005743-29580d3d-a682-491f-a1cb-db4495f583c4.png">

**Comment by timoyates** [comment, marked as answer]
Hi @oguzgelal, thanks for the feedback! Reordering views via drag-and-drop is coming soon 😊 If you'd like to workaround in the meantime, you can duplicate the other two views and delete the original ones. Cheers!

**Reply by oguzgelal** [reply]
@timoyates Thank you! Loving the new Issues ❤️
